Dronuts/client_src/src/components/employee/more_components/Drone.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component} from 'react';
```

Comment:
In general, the code in this pull request looks good. I do wonder why you chose to have two files of components for drones on the employee side in more_components. Can you explain your reasoning?

Reply:
I was having difficulty importing Drone.js and Order.js into the Drones.js and Home.js files. Somehow separating them into a different directory helped to successfully import them.